server/controllers/offuserController.js

```diff
@@ -84,6 +86,77 @@ const deleteUserById = async (req, res) => {
   }
 };
 
+const sendEmail = async (req, res) => {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    host: 'smtp.gmail.com',
+    port: 587,
+    secure: false,
+    auth: {
+      user: 'kennethzwan@gmail.com',
+      pass: process.env.GMAIL_PASSWORD,
```

Comment:
could we get this un-hardcoded? I think it should be pretty simple (making the email a part of the req.body/req.params)
and i'll continue testing this after that change 🤩